sources/packages/backend/apps/db-migrations/src/sql/StudentAssessments/Add-related-application-assessment-id-col.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+  sims.student_assessments
+ADD
+  COLUMN related_application_assessment_id INT REFERENCES sims.student_assessments(id);
```

Comment:
Do we need to have this column defaulted to NULL for the first one without having any related applications?

Reply:
I got the point but the value will be inserted as null when not provided 😉